client/layout/index.js

```diff
@@ -27,6 +27,12 @@ import StoreAlerts from './store-alerts';
 class Layout extends Component {
 	componentDidMount() {
 		this.recordPageViewTrack();
+
+		const path = this.getPath();
+
+		if ( 'dashboard' !== path ) {
+			document.body.classList.remove( 'woocommerce-profile-wizard__body' );
```

Comment:
Would it be possible to add a check in `add_admin_body_classes()` to see if we're in the dashboard page before adding this class?  That way we can remove these 3 lines and keep this components scope down.

Reply:
I have a check in there that checks if we are on a JS page, but the problem is we can't tell which wc-admin route they on because the route is stored in a JS hash that is not passed down to the server.

I did a little exploration to see what we can do, but switching to another routing method / URL structure would be non-trivial. I think we might have to stick with this for now. Does that seem OK?

Reply:
https://github.com/woocommerce/woocommerce-admin/issues/2267

Reply:
Ya, that seems very reasonable and thanks for your research around the issue!  If we make those routing changes you suggested at a later date, we can revisit this.

Reply:
I ended up generalizing this a bit more in the meantime. I now have it adding a `woocommerce-admin-is-loading` class.

I'm still only appending it when necessary, but this allows us to do something similar on other routes in the future (like future Woo App pages). It also allows me to drop the `dashboard` check here. We'll just always remove the class once `<Layout />` has loaded, and the  profiler will append its own profiler class to continue hiding elements.